sonic-pcied/scripts/pcied

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python2
+
+"""
+    pcied
+    PCIe device monitoring daemon for SONiC
+"""
+
+try:
+    import os
+    import sys
+    import signal
+    import threading
+    import subprocess
+    from sonic_daemon_base.daemon_base import Logger
+    from sonic_daemon_base.daemon_base import DaemonBase
+    import swsssdk
+except ImportError, e:
+    raise ImportError (str(e) + " - required module not found")
+
+#
+# Constants ====================================================================
+#
+SYSLOG_IDENTIFIER = "pcied"
+
+PCIE_RESULT_REGIX = "PCIe Device Checking All Test"
+PCIE_TABLE_NAME = "PCIE_STATUS"
+
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
+PCIE_CONF_FILE = 'pcie.yaml'
+SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
+HWSKU_KEY = 'DEVICE_METADATA.localhost.hwsku'
+PLATFORM_KEY = 'DEVICE_METADATA.localhost.platform'
+
+PCIED_MAIN_THREAD_SLEEP_SECS = 60
+REDIS_HOSTIP = "127.0.0.1"
+
+# Global logger class instance
+logger = Logger(SYSLOG_IDENTIFIER)
+
+#
+# Helper Function ==============================================================
+#
+
+def get_platform_and_hwsku():
```

Comment:
This is a common function. Can you please import from the UtilHelper class?

https://github.com/Azure/sonic-utilities/blob/master/utilities_common/util_base.py#L73